.github/workflows/installer.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: ./install.sh
 
       - name: Test code-server
-        run: yarn test:standalone-release code-server
+        run: CODE_SERVER_PATH="code-server" yarn test:integration
```

Comment:
Since we install code-server in these steps, we need to tell the integration tests to look at `code-server` and use the CLI/binary.